frontend/src/components/HacknightsParticipants.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-combobox
+          v-model="selectedParticipants"
+          outlined
+          :items="
+            getParticipants.filter(
+              participant =>
+                !getHacknight.participants.filter(y => y.id === participant.id)
+                  .length
+            )
+          "
+          item-text="email"
+          item-value="email"
+          :search-input.sync="search"
+          hide-selected
+          label="Add participants"
+          multiple
+          persistent-hint
+          small-chips
+          clearable
+          deletable-chips
+          append-outer-icon="mdi-cloud-upload"
+        >
+          <template v-if="noData" v-slot:no-data>
+            <v-list-item>
+              <v-list-item-content>
+                <v-list-item-title>
+                  No results matching "<strong>{{ search }}</strong
+                  >".
+                </v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+          </template>
+          <template v-slot:append-outer>
+            <v-btn
+              color="blue-grey"
+              class="ma-2 white--text add-hacknight-btn"
+              @click="onAddParticipants"
+              offset-y
+            >
```

Comment:
When you try to send empty list 400 error occurs. Maybe we should validate here?

Reply:
I added disabled prop based on whenever list of participants is empty or not